gapi/eth.go

```diff
@@ -0,0 +1,45 @@
+package gapi
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/RTradeLtd/config"
+	"github.com/RTradeLtd/gapimit/request"
+	"github.com/RTradeLtd/gapimit/response"
+	pb "github.com/RTradeLtd/gapimit/service"
```

Comment:
naming on this package seems a bit iffy - calling the repo `github.com/RTradeLtd/temporal-grpc` and calling the package `temporal` might be more useful:

- namespace for package `temporal` is unused
- it makes sense for other services using the gRPC client to call `temporal.NewClient`
- having MIT in the repo name doesnt seem like a good use of name haha

even better, could we just call the repo `RTradeLtd/grpc`? and then the temporal gRPC client could be in in `github.com/RTradeLtd/grpc/temporal`. This allows us to use the repo for other purposes as well:

```
github.com/RTradeLtd/grpc/temporal
github.com/RTradeLtd/grpc/ipfs-orchestrator
github.com/RTradeLtd/grpc/payment
github.com/RTradeLtd/grpc/lens
```

thoughts?

Reply:
Haha I entirely agree, I published that package on git very late at night and couldn't think of something else :joy: